site/tests/support/conftest.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# invenio-administration is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
```

Comment:
Is this file needed? If no fixtures are setup, the conftest is not needed

Reply:
it was a leftover, thanks!